pkg/stis/calstis/cs1/dodqi.c

```diff
@@ -652,6 +653,8 @@ int doppmin, doppmax  i: offsets for Doppler shift
 	    xlength = 2;
 	    ylength = tabrow->length * 2;
 	}
+	else
+	    assert(0);
```

Comment:
I don't know about this. The standard practice in HSTCAL is to set `status` to the appropriate error code and then return it (which will result in HSTCAL terminating with some error code), and also to write the error message to trailer file.